1. Line 9 prints "values added: 20".
2. Line 13 prints "final result: 20".
3. Line 9 prints "values added: 20". 
4. Line 13 returns an error. It is because the when we declare the variable let, the result can only be accesed within the block it is defined in. In this case, the result can be accessed inside the if function. 
5. Line 9 returns an error. It is because we cannot reassign after it is assigned for the first time. 
6. Line 10 returns an error. It is because we cannot reassign after it is assigned for the first time. 
   
   